db/migrate/20250617132607_add_guardian_fields_to_user_compliance_info.rb

```
@@ -0,0 +1,8 @@
+class AddGuardianFieldsToUserComplianceInfo < ActiveRecord::Migration[7.0]
```

RuboCop (1.75.5)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)